scripts/freeze-cabal-to-stack-resolver.sh

```diff
@@ -2,8 +2,15 @@
 
 set -euxo pipefail
 
+# must remove the prior freeze file to actually update
+rm -f cabal.project.freeze
+
 stack --test ls dependencies \
     | sed -e 's/^\([a-zA-Z0-9.-]*\) \([0-9.]*\)/--constraint="\1 == \2"/' \
     | xargs -x cabal freeze --enable-tests
 
+# ghc-bignum is removed from the freeze file because it is wired into
+# GHC and creates problems when nix builds tools with different GHCs
+sed -i -e '/any\.ghc-bignum/d' cabal.project.freeze
```

Comment:
we should add alex and happy here as well, because there never seems to be the exact version of these packages in the nix version of cabal